src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx

```diff
@@ -462,7 +433,12 @@ const UniqueTokenExpandedStateHeader = ({
     <Stack space="15px (Deprecated)">
       <Columns space="24px">
         <Heading containsEmoji color="primary (Deprecated)" size="23px / 27px (Deprecated)" weight="heavy">
-          {buildUniqueTokenName(asset)}
+          {buildUniqueTokenName({
+            collectionName: asset.collectionName ?? '',
+            tokenId: asset.tokenId,
+            name: asset.name ?? '',
```

Comment:
super nit but in general || works unless you specifically need to handle falsy values